lib/model/map/ObjectTableMap.php

```diff
@@ -49,16 +49,16 @@ public function initialize()
 	 */
 	public function buildRelations()
 	{
-    $this->addRelation('accession', 'accession', RelationMap::ONE_TO_ONE, array('id' => 'id', ), 'CASCADE', null);
```

Comment:
Nitpick: some of these lines just changed positions, but doesn't matter much.

Reply:
Thanks @mcantelon.  I definitely missed that some of these were re-ordered.  I have fixed these in a new commit.  The end result when I squash these commits will be that some of these files will end up being unchanged and disappear.

Reply:
Based on our discussion below, I have reverted my fix to re-order these items.